<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionsPage</class>
 <widget class="QWidget" name="OptionsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#OverviewPage { color: #0C3407; } QToolTip { border: 1px solid black; background-color: #0C3407; color: #FFFFFF; }</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>These options let you customize Sapience AIFX behavior.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="dataPlumesCheckBox">
       <property name="text">
        <string>Data Plumes Enabled</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="enableStakingCheckBox">
       <property name="text">
        <string>Staking Enabled</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="voiceEnabledCheckBox">
       <property name="text">
        <string>Voice Enabled</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
